package.json

```diff
@@ -10,11 +10,11 @@
     },
     "devDependencies": {
         "babel-eslint": "^8.2.6",
-        "eslint": "^5.0.0",
+        "eslint": "^6.0.0",
         "mocha": "^5.2.0"
     },
     "peerDependencies": {
-        "eslint": "^5"
+        "eslint": "^6"
```

Comment:
Based on the changes applied in this pull request it looks that this implementation could be actually compatible with both eslint 5 and eslint 6.

Given that we also bumping the package version as a new patch release, how about setting the peerDependencies to "^5 || ^6"?

This would also help with upgrading this dependency in the addons-linter repository, as it does depend on eslint 5 (and we haven't ported it yet to the eslint 6 API).

Reply:
:ok_hand: commit incoming